src/createServer.js

```diff
@@ -1,3 +1,71 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  return http.createServer((req, res) => {
+    const [path, queryString] = req.url.split('?');
+    const errors = [];
+
+    if (!path || path === '/') {
+      errors.push({
+        message:
+          // eslint-disable-next-line max-len
+          'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    }
```

Comment:
The check for `!path || path === '/'` is redundant because `textToConvert` is derived from `path.slice(1)`, which will be an empty string if `path` is '/'. Therefore, the check on line 19 already covers this case. You can remove this block to simplify the code.